statstracker/src/tests/App.test.js

```diff
@@ -0,0 +1,101 @@
+// Required to get <App /> running:
+import React from 'react';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import rootReducer from '../reducers/reducers';
+import App from '../components/App';
+const store = createStore(rootReducer);
+
+import { render, fireEvent } from 'react-testing-library';
+import 'jest-dom/extend-expect';
+
+const packagedApp = (
+  <Provider store={store}>
+    <App />
+  </Provider>
+);
+
+describe('<App />', () => {
+  it('renders successfully', () => {
+    render(packagedApp);
+  });
+});
+
+describe('Strike Button', () => {
+  it('adds a strike 0 to 1', () => {
+    const { getByText } = render(packagedApp);
+    const button = getByText(/strike$/i);
+    fireEvent.click(button);
+    getByText(/strikes: 1/i);
+  });
+
+  it('resets count on 3rd strike', () => {
+    const { getByText } = render(packagedApp);
+    const button = getByText(/strike$/i);
+    fireEvent.click(button);
+    fireEvent.click(button);
+    getByText(/balls: 0/i);
+    getByText(/strikes: 0/i);
```

Comment:
continuing from last comment -- if no buttons actually are linked up to their click handlers and never fire the desired funciton, this test would still pass because the desired end state is identical to the default state. insert getByText tests in between the fireEvents to verify the behavior at multiple points